src/Table.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { mapToCssModules } from './utils';
+import { mapToCssModules, deprecated } from './utils';
 
 const propTypes = {
   className: PropTypes.string,
   cssModule: PropTypes.object,
   size: PropTypes.string,
   bordered: PropTypes.bool,
   striped: PropTypes.bool,
-  inverse: PropTypes.bool,
+  inverse: deprecated(PropTypes.bool, 'Please use the prop "dark"'),
+  dark: PropTypes.bool,
   hover: PropTypes.bool,
   reflow: PropTypes.bool,
-  responsive: PropTypes.bool,
+  responsive: PropTypes.oneOfType([PropTypes.bool, PropTypes.oneOf(['sm', 'md', 'lg', 'xl'])]),
```

Comment:
I would just allow a string in case someone wants to extend bootstrap to add additional/custom breakpoints 
```
PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
```